src/library/scala/collection/immutable/WrappedString.scala

```diff
@@ -73,18 +73,6 @@ final class WrappedString(private val self: String) extends AbstractSeq[Char] wi
       case _       => super.lastIndexOf(elem, end)
     }
 
-  override def indexOfSlice[B >: Char](that: collection.Seq[B], from: Int = 0): Int =
```

Comment:
Nice catch. We should put `import scala.Predef.{wrapString => _, _}` into this file to avoid such accidents. `ArrayOps` does the same for the array conversions.